src/lib/auth.js

```diff
@@ -79,6 +81,6 @@ function findUser(profile, accessToken, config, callback)  {
 }
 
 function verifyApiKey(config, req)  {
-    var apiKey = req.headers['apikey'] || req.query['apikey'] || '';
-    return (config.apiKeys.indexOf(apiKey) > -1)
+    var apiKey = req.headers['authorization'] || '';
```

Comment:
Is this case-insensitive? ie. will it accept `Authorization` as well as `authorization`?